src/proxy-ui-api/frontend/src/store/modules/client.ts

```diff
@@ -210,6 +202,33 @@ export const actions: ActionTree<ClientState, RootState> = {
 
   },
 
+  registerClient({ commit, state }, { instanceId, memberClass, memberCode, subsystemCode }) {
+    const clientId = instanceId + ':' + memberClass + ':' + memberCode + ':' + subsystemCode;
+    return axios.put(`/clients/${clientId}/register`, {});
```

Comment:
It feels a bit inconsistent that we have here 3 methods returning a promise and one of them having error handling but the two others do not. I wonder if we could come a more consistent way?

Reply:
Fixed